src/APIFetch.js

```diff
@@ -30,3 +10,31 @@ export const getMovies = () => {
     }
   })
 }
+
+export const getSelectedMovie = (id) => {
+  const selectedMovieDetails = fetch(`${movieApi}/movies/${id}`)
+      .then(response => {
+        if (response.ok) {
+          return response.json()
+        } else {
+          throw new Error(`Uh oh! We can't find the movie you're looking for!`)
+        }
+      })
+
+  const selectedMovieTrailer = fetch(`${movieApi}/movies/${id}/videos`)
+    .then(response => {
+      if (response.ok) {
+        return response.json()
```

Comment:
The first block of each fetch look similar to one another - have you considered just sending the whole URL through as an argument - might let you condense the many fetches down into one